app/scripts/components/common/page-header/default-config.ts

```diff
@@ -8,44 +18,125 @@ import {
   ABOUT_PATH
 } from '$utils/routes';
 
-let defaultMainNavItems:(ExternalNavLink | InternalNavLink | DropdownNavLink | ModalNavLink)[] = [{
-  title: 'Data Catalog',
-  to: DATASETS_PATH,
-  type: NavItemType.INTERNAL_LINK
-}, {
-  title: 'Exploration',
-  to: EXPLORATION_PATH,
-  type: NavItemType.INTERNAL_LINK
-}, {
-  title: getString('stories').other,
-  to: STORIES_PATH,
-  type: NavItemType.INTERNAL_LINK
-}];
-
-if (!!process.env.HUB_URL && !!process.env.HUB_NAME) defaultMainNavItems = [...defaultMainNavItems, {
-  title: process.env.HUB_NAME,
-  href: process.env.HUB_URL,
-  type: NavItemType.EXTERNAL_LINK
-} as ExternalNavLink];
-
-let defaultSubNavItems:(ExternalNavLink | InternalNavLink | DropdownNavLink | ModalNavLink)[] = [{
-  title: 'About',
-  to: ABOUT_PATH,
-  type: NavItemType.INTERNAL_LINK
-}];
+let defaultMainNavItems: (
+  | ExternalNavLink
+  | InternalNavLink
+  | DropdownNavLink
+  | ModalNavLink
+)[] = [
+  {
+    title: 'Data Catalog',
+    to: DATASETS_PATH,
+    type: NavItemType.INTERNAL_LINK
+  },
+  {
+    title: 'Exploration',
+    to: EXPLORATION_PATH,
+    type: NavItemType.INTERNAL_LINK
+  },
+  {
+    title: getString('stories').other,
+    to: STORIES_PATH,
+    type: NavItemType.INTERNAL_LINK
+  }
+];
+
+if (!!process.env.HUB_URL && !!process.env.HUB_NAME)
+  defaultMainNavItems = [
+    ...defaultMainNavItems,
+    {
+      title: process.env.HUB_NAME,
+      href: process.env.HUB_URL,
+      type: NavItemType.EXTERNAL_LINK
+    } as ExternalNavLink
+  ];
+
+let defaultSubNavItems: (
+  | ExternalNavLink
+  | InternalNavLink
+  | DropdownNavLink
+  | ModalNavLink
+)[] = [
+  {
+    title: 'About',
+    to: ABOUT_PATH,
+    type: NavItemType.INTERNAL_LINK
+  }
+];
 
 if (process.env.GOOGLE_FORM) {
-  defaultSubNavItems = [...defaultSubNavItems, {
-    title: 'Contact us',
-    src: process.env.GOOGLE_FORM,
-    type: NavItemType.MODAL
-  }];
+  defaultSubNavItems = [
+    ...defaultSubNavItems,
+    {
+      title: 'Contact us',
+      src: process.env.GOOGLE_FORM,
+      type: NavItemType.MODAL
+    }
+  ];
 }
 
-const mainNavItems = getNavItemsFromVedaConfig()?.mainNavItems?? defaultMainNavItems;
-const subNavItems = getNavItemsFromVedaConfig()?.subNavItems?? defaultSubNavItems;
+const defaultFooterSettings = {
+  secondarySection: {
+    title: 'email test',
+    to: '/data-catalog',
+    type: 'Email'
+  },
+  returnToTop: true
+};
+
+const defaultFooterPrimaryContactItems = [
```

Comment:
do we want the footer items to be different from the header? In my understanding, it was just a repetition of the main nav items - which would mean we should use the same configuration that we base the header on.

Reply:
As I was working through this, this question came up and I think we should discuss further. My thoughts are that  we can and should model the footer materials after the header data model. But the instances are going to request for individualized configurability of the footer so they can customize past the header navigation. What are your thoughts?

Reply:
This sounds like a good case for [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) - let's wait for that request to come in first :)